golang/cli/vout.go

```diff
@@ -18,9 +18,24 @@ func prettifyOutput(
 	// Get the output type in readable format
 	outputTypeString := btcspv.GetOutputType(outputType)
 
+	// Get the address associated with the output
+	var address string
+	switch outputType {
```

Comment:
Can we move this to a separate function?

Reply:
Definitely, should I export it?

Reply:
Ok, I just added it as an internal function.  Figured we could easily export it if we decide it would be useful to be accessible elsewhere.